ui/src/views/AutogenView.vue

```diff
@@ -858,6 +858,10 @@ export default {
         delete params.showunique
       }
 
+      if (['listVirtualMachinesMetrics'].includes(this.apiName) && this.dataView) {
+        delete params.details
```

Comment:
we remove the details are it may not be `all` as set for the list view (tabular view)